content/en/hosting/monitoring/setup.md

```diff
@@ -141,19 +141,36 @@ With the [release of 1.1.0](https://github.com/medic/cht-watchdog/releases/tag/1
 
 {{% alert title="Note" %}}
 Always run this longer version of the `docker compose` command which specifies both compose files for all future [upgrades](#upgrading).
+This configuration also works for [couch2pg]({{< relref "apps/tutorials/couch2pg-setup" >}} data. Edit `sql_servers.yml` to set the value of `collectors` to `couch2pg`.
 {{% /alert %}}
 
-#### couch2pg Data (Remote)
+#### CHT Sync Data (Remote)
 
-While not the default setup, and not what most deployments need, you may want to set up a way to monitor couch2pg data without sharing any Postgres credentials. Instead of sharing credentials, you expose an HTTP endpoint that requires no login or password.  Of course, similar to  CHT Core's [Monitoring API]({{< relref "apps/reference/api#get-apiv2monitoring" >}}), this endpoint should be configured to not share sensitive information (since it will be publicly accessible).
+While not the default setup, and not what most deployments need, you may want to set up a way to monitor CHT Sync data without sharing any Postgres credentials. Instead of sharing credentials, you expose an HTTP endpoint that requires no login or password.  Of course, similar to  CHT Core's [Monitoring API]({{< relref "apps/reference/api#get-apiv2monitoring" >}}), this endpoint should be configured to not share sensitive information (since it will be publicly accessible).
+
+If you are running CHT Sync using Kubernetes:
+1. Uncomment the `metrics_exporter` value in your `values.yaml` file and make sure `enabled` is set to `true`.
+1. Expose the metrics endpoint to the internet. This can be done by setting the `service.type` to `LoadBalancer` or `NodePort` in your `values.yaml` file.
+1. On your watchdog instance, create a custom scrape definition file: `cp exporters/postgres/scrape.yml ./exporters/postgres/scrape-custom.yml`
+1. Edit `scrape-custom.yml` so that it has the ip address of `SQL_EXPORTER_IP` from step 2 above.  If that was `10.220.249.15`, then you file would look like:
+   ```yaml
+    scrape_configs:
+      - job_name: sql_exporter
+        static_configs:
+          - targets: ['10.220.249.15:9399']
+   ```
+1. Finally, on your watchdog instance, start (or restart) your server including the `compose.scrape-only.yml` compose file:
+    ```bash
+    docker compose --env-file .env  -f exporters/postgres/compose.yml -f exporters/postgres/compose.scrape-only.yml up -d
+    ```
```

Comment:
I think it would be cleaner to break this up to be either k8s based (CHT Sync only) or docker based (both Sync and couch2pg).  



```suggestion
This section has two steps. The first is to expose the password-less metrics endpoint and the second is to scrape it with Prometheus.   Here's documentation on how to set up a Kubernetes endpoint only for CHT Sync or a Docker endpoint for either CHT Sync or couch2pg. 

##### Postgres exporter on Kubernetes (CHT Sync only) 

1. Uncomment the `metrics_exporter` value in your `values.yaml` file and make sure `enabled` is set to `true`.
1. Expose the metrics endpoint to the internet. This can be done by setting the `service.type` to `LoadBalancer` or `NodePort` in your `values.yaml` file.

Continue on to [set up the scrape on Watchdog](#set-up-watchdog-to-scrape-the-sql-exporter).
```